app/templates/organization/settings.hbs

```diff
@@ -0,0 +1,113 @@
+<section class='organization-settings'>
+  {{#if (and features.proVersion organization.permissions.admin)}}
+    <section class="active-visiblity-settings" data-test-active-visiblity-settings>
+      <h3>Active Visibility Settings</h3>
+      <div class="insights-settings">
+        <div class="name" data-test-insights-org-settings-title>
+          Insights
+        </div>
+        <p class="description" data-test-insights-org-settings-description>
+          Make more informed decisions about your development workflow using your build Insights.<br />
+          {{#link-to 'owner.repositories' organization.login (query-params tab='insights')}}View {{organization.name}}'s Insights{{/link-to}}
+        </p>
+        <ul class='travis-radio-list'>
+          <li class='travis-radio-list__item'>
+            <label for="organization_insight_admins">Only allow organization owners to see insights from your private builds</label>
+            <input
+              type="radio"
+              name="organization_insight_vis"
+              class="travis-radio-input"
+              id="organization_insight_admins"
+              value="admins"
+              checked={{eq privateInsightsVisibility 'admins'}}
+              onchange={{action (mut privateInsightsVisibility) 'admins'}}
+            />
+            <label for="organization_insight_admins" class="travis-radio-label"></label>
+          </li>
+          <li class='travis-radio-list__item'>
+            <label for="organization_insight_members">Only allow organization members to see insights from your private builds</label>
+            <input
+              type="radio"
+              name="organization_insight_vis"
+              class="travis-radio-input"
+              id="organization_insight_members"
+              value="members"
+              checked={{eq privateInsightsVisibility 'members'}}
+              onchange={{action (mut privateInsightsVisibility) 'members'}}
```

Comment:
hah well re my `toggle` commentary, I see that you’re mutating in-template here, so you already know about that… good call here!